include/prophesee_ros_stereo_driver/prophesee_ros_stereo_publisher.h

```diff
@@ -0,0 +1,130 @@
+/*******************************************************************
+ * File : prophesee_ros_publisher.h                                *
+ *                                                                 *
+ * Copyright: (c) 2015-2019 Prophesee                              *
+ *******************************************************************/
+
+#ifndef STEREO_PUBLISHER_H_
+#define STEREO_ROS_PUBLISHER_H_
```

Comment:
some inconsistency here.
should be #ifndef STEREO_ROS_PUBLISHER_H_

Reply:
Fixed, thanks.